qiskit_braket_provider/providers/braket_job.py

```diff
@@ -77,26 +78,26 @@ def _get_result_from_aws_tasks(
     return experiment_results
 
 
-class AWSBraketJob(JobV1):
-    """AWSBraketJob."""
+class AmazonBraketTask(JobV1):
+    """AmazonBraketTask."""
 
     def __init__(
         self,
-        job_id: str,
+        task_id: str,
         backend: BackendV2,
         tasks: Union[List[LocalQuantumTask], List[AwsQuantumTask]],
-        **metadata: Optional[dict]
+        **metadata: Optional[dict],
     ):
-        """AWSBraketJob for local execution of circuits.
+        """AmazonBraketTask for local execution of circuits.
 
         Args:
-            job_id: id of the job
+            task_id: id of the task
             backend: Local simulator
             tasks: Executed tasks
             **metadata:
         """
-        super().__init__(backend=backend, job_id=job_id, metadata=metadata)
-        self._job_id = job_id
+        super().__init__(backend=backend, task_id=task_id, metadata=metadata)
```

Comment:
Is there a task_id attribute in JobV1 class? What happens when a user calls `.job_id()` on an object of this class?

Reply:
Hi, there is no `task_id` attribute in the JobV1 class actually, but we have job_id(). So  actually the argument name should be ` job_id`. What I am thinking is having 
```
       super().__init__(backend=backend, job_id=job_id, metadata=metadata)
    ```
but it would be  a bit cofusing

Reply:
was able to resolve this in the latest commit https://github.com/qiskit-community/qiskit-braket-provider/pull/105/commits/edaba8d167f8a734e845b6c774cae4174c9f2223